### ISSUE TITLE
hab pkg build -D on windows requiring TTY

### ISSUE THREAD

**Issue by wduncanfraser** [habitat-sh/habitat#6639]
After updating our windows CI workers to hab 0.79.1 from 0.72.0 doing a `hab pkg build -D` is erroring out with

```
Status: Downloaded newer image for habitat-docker-registry.bintray.io/win-studio-x86_64-windows:0.79.1
the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'
```

0.79.1 builds fine in CI with windows studios, and rolling back to 0.72.0 fixed the issue with docker studios.

OS: Server 2016 Core

**Comment by mwrock** [comment]
That got fixed in 0.81.0. To work around this in 0.79.1 you need to use `hab studio build -D --no-tty --non-interactive <PATH>`

**Comment by wduncanfraser** [comment]
@mwrock Thanks!
